exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/InnerTableLink.kt

```diff
@@ -28,6 +30,7 @@ class InnerTableLink<SID : Comparable<SID>, Source : Entity<SID>, ID : Comparabl
     val target: EntityClass<ID, Target>,
     _sourceColumn: Column<EntityID<SID>>? = null,
     _targetColumn: Column<EntityID<ID>>? = null,
+    additionalColumns: List<Column<*>>? = null,
```

Comment:
I see that this parameter is a way to redefine default list of extra columns (`(table.columns - sourceColumn - targetColumn)` with some filters), but it's not used now anywhere, probably we could a test for it also.

I don't expect that something bad could happen, but what if user add in this list mentioned `sourceColumn` or `targetColumn`.